hbase-common/src/main/java/org/apache/hadoop/hbase/io/crypto/tls/X509Util.java

```diff
@@ -80,6 +80,13 @@ public final class X509Util {
   private static final String TLS_ENABLED_PROTOCOLS = CONFIG_PREFIX + "enabledProtocols";
   private static final String TLS_CIPHER_SUITES = CONFIG_PREFIX + "ciphersuites";
 
+  public static final String TLS_CONFIG_VERIFY_CLIENT_HOSTNAMES =
+    CONFIG_PREFIX + "verify.client.hostnames";
+  public static final String TLS_CONFIG_VERIFY_SERVER_HOSTNAMES =
+    CONFIG_PREFIX + "verify.server.hostnames";
+
+  public static final String TLS_CONFIG_CLIENT_AUTH_MODE = CONFIG_PREFIX + "client.auth.mode";
```

Comment:
nit: the "CONFIG" word is not part of other property constants, so it's not consistent atm.